packages/server/src/api/controllers/query/import/sources/curl.ts

```diff
@@ -1,10 +1,9 @@
 import { ImportSource, ImportInfo } from "./base"
 import { Query } from "../../../../../definitions/common"
 import { URL } from "url"
-
 const curlconverter = require("curlconverter")
```

Comment:
Its probably worth throwing a comment in here to explain the pain of trying to convert curlconverter to `import`!